### PR TITLE
Cbs tests

### PR DESCRIPTION
**Pull request by BjoerklundJacob**

### CODE REVIEW

File: Simulation/Pathfinding/ConstraintTree/ConstraintTree.cpp [hunk 1]
```diff
@@ -26,7 +26,7 @@ void ConstraintTree::setSolution(Solution s){
 }
 void ConstraintTree::setSolution(std::vector<Path> paths, std::vector<AgentInfo> agents){
```
Comment (Simson475) [inline Simulation/Pathfinding/ConstraintTree/ConstraintTree.cpp:33]: yeet

File: Simulation/Pathfinding/HighLevelCBS.cpp [hunk 2]
```diff
@@ -24,6 +24,9 @@ Solution HighLevelCBS::findSolution(std::shared_ptr<Graph> graph, std::vector<Ag
     int iterations = 0;
     while (open.size() > 0) {
         iterations++;
+        if (iterations == 3){
```
Comment (Suprezer) [inline Simulation/Pathfinding/HighLevelCBS.cpp]: This probably needs to go, far far away.

**Review by Zzeeker — APPROVED** [review]

File: Simulation/Pathfinding/HighLevelCBS.cpp (hunk 2 above)
Comment (Simson475) [inline Simulation/Pathfinding/HighLevelCBS.cpp]: GIT GONE

**Review by Suprezer — COMMENTED** [review]
We like tests

File: Simulation/Pathfinding/ConstraintTree/ConstraintTree.cpp (hunk 1 above)
Comment (BjoerklundJacob) [reply Simulation/Pathfinding/ConstraintTree/ConstraintTree.cpp:33]: turns out, den bliver brugt, så har fixed det nu

**Review by Suprezer — APPROVED** [review]
Looking fine

**Review by Simson475 — APPROVED** [review]

**Pull request merged**